source/features/rename-diff-settings-to-icon.tsx

```diff
@@ -0,0 +1,25 @@
+/*
+Renames Diff Settings to be an icon instead
+*/
+
+import React from 'dom-chef';
+import select from 'select-dom';
+import features from '../libs/features';
+import {gear} from '../libs/icons';
+
+async function init() {
+	const diffSettings = select('.pr-review-tools .diffbar-item summary');
+	diffSettings.innerText = '';
+	diffSettings.append(<span class="data-menu-button">{gear()}</span>);
```

Comment:
What is `data-menu-button` for?

Also
```suggestion
	diffSettings.append(
		<span class="data-menu-button">{gear()}</span>,
		' '
	);
```

Reply:
I copied the span/svg from another spot and the `span` was making it dark instead of grey, you're right and it wasn't needed though, I should have looked at that first. :)